python/lsst/meas/astrom/match.py

```diff
@@ -0,0 +1,102 @@
+from __future__ import absolute_import, division, print_function
+
+from lsst.pex.config import Config, Field, ConfigurableField
+from lsst.pipe.base import Task, Struct
+from . import MatchMetadata, LoadAstrometryNetObjectsTask
+import lsst.afw.table as afwTable
+import lsst.afw.coord as afwCoord
+from lsst.afw.geom import arcseconds
+
+
+__all__ = ["MatchConfig", "MatchTask"]
+
+
+class MatchConfig(Config):
+    """Configuration for MatchTask"""
+    refObjLoader = ConfigurableField(target=LoadAstrometryNetObjectsTask, doc="Load reference objects")
+    matchRadius = Field(dtype=float, default=0.25, doc="Matching radius, arcsec")
+
+
+class MatchTask(Task):
+    """!Simple matching to a reference catalog using the existing astrometric solution
+
+    @anchor MatchTask_
+
+    @section meas_astrom_match_Contents Contents
+
+     - @ref meas_astrom_match_Purpose
+     - @ref meas_astrom_match_Initialize
+     - @ref meas_astrom_match_IO
+     - @ref meas_astrom_match_Config
+     - @ref meas_astrom_match_Example
+
+    @section meas_astrom_match_Purpose  Description
+
+    Match sources to reference objects. This is often done as a preliminary step to fitting an astrometric
+    or photometric solution. For details about the matching algorithm see match.h
```

Comment:
Please say a bit more about how this differs from our usual matcher, e.g. that it performs no scaling or rotation. Otherwise I worry that users will get confused about what this does vs the other matcher(s).

The reference to match.h is a bit opaque (and the case is wrong). I suggest something like "The matching algorithm is matchRaDec, which is defined in lsst/afw/table/Match.h".